imod/msw/copy_files.py

```diff
@@ -0,0 +1,55 @@
+from pathlib import Path
+from shutil import copy2
+from typing import cast
+
+import numpy as np
+import xarray as xr
+
+from imod.logging import logger
+from imod.logging.loglevel import LogLevel
+from imod.msw.pkgbase import MetaSwapPackage
+from imod.typing import Imod5DataDict
+
+_LOG_MESSAGE_TEMPLATE = """\
+Will not copy files {filtered}, these will be generated by iMOD Python
+instead."""
+
+
+class CopyFiles(MetaSwapPackage):
+    def __init__(self, paths: list[str]):
+        super().__init__()
+        paths_da = xr.DataArray(
+            paths, coords={"file_nr": np.arange(len(paths))}, dims=("file_nr",)
+        )
+        self.dataset["paths"] = paths_da
+
+    @classmethod
+    def from_imod5_data(cls, imod5_data: Imod5DataDict):
+        paths = cast(list[list[str]], imod5_data["extra"]["paths"])
+        paths_unpacked = {Path(p[0]) for p in paths}
```

Comment:
Why are you using curly brackets and not block brackets? I thought curly brackets are only for dictionaries, but this seems to be a list.
The same thing occurs on line 39

Reply:
This is a Python "set". They are very useful to check values in iterables for uniqueness. And if one set contains values not present, or present, in another set. For docs, see: https://docs.python.org/3/library/stdtypes.html#set

I wanted to make this more explicit by doing ``set([Path(p[0]) for p in paths])``, but Ruff disagreed with me. I guess the Ruff devs think the set is common knowledge as it is a Python builtin (though a bit more niche).